### PR TITLE
Size Added

### PR DESCRIPTION
**Pull request by poshan0126**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by saarques** [comment]
Hey @poshan0126 Your files seems to be alright but you must get rid of this conflict before the merging team approaches.

File: _data/poshan0126/poshan0126.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Poshan Pandey # No longer than 28 characters
+institution: Nepal College of Information Technology 🚩 # no longer than 58 characters
+profile_pic: poshan.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```
Comment (Mohammed-Shoaib) [inline _data/poshan0126/poshan0126.md:4]: Incorrect file name, please do one of the following:
1. Change `poshan.jpg` to `git.jpg` in the markdown file.
2. Rename `git.jpg` to `poshan.jpg`.

**Comment by poshan0126** [comment]
I already deleted the old image sir!

**Comment by Mohammed-Shoaib** [comment]
@poshan0126 You deleted the old image from your pull request but not the image that was merged in #3978. I meant [this](https://github.com/education/graduation/blob/master/_data/poshan0126/poshan.jpg) image is causing the conflict. Please _carefully_ **read** the instructions that I mentioned in the above comment.

**Pull request closed without merging**